lib/ReactViews/BottomDock/chart-disclaimer.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../Sass/common/variables";
+@import "../../Sass/common/mixins";
+
+.chart-disclaimer-panel {
+  background: #9a4b4b;
+  color: white;
+  padding: 1px 10px;
```

Comment:
Let's use the $padding from `_varaibles.scss` in place of the 10px

Reply:
Done